Yank/repex.py

```diff
@@ -1609,6 +1709,8 @@ def run(self, n_iterations=None):
         n_iterations : int, optional
            If specified, only at most the specified number of iterations
            will be run (default is None).
+        continue_past_online_target : bool, optional, default False
```

Comment:
Do we need this extra argument? Maybe we can just set `online_analysis_target_error` or `online_analysis_target_error` to `None`, or use some of the existing options anyway?